documentation/modules/ROOT/pages/connectors/oracle.adoc

```diff
@@ -3299,6 +3299,44 @@ endif::community[]
 When connecting to Oracle using mutual TLS authentication (mTLS), this involves both the connector and the database service proving their identities.
 The {prodname} for Oracle connector relies on the Oracle JDBC driver's built-in functionality to support mTLS authentication.
 
+There are two ways to establish mTLS connections:
+
+* xref:#using-mtls-secure-connections-java-key-trust-stores[]
```

Comment:
```suggestion
* xref:#using-mtls-secure-connections-java-key-trust-stores[Using Java key and trust stores to configure mTLS connections].
```

Reply:
In shared content (community/product), we must provide link text, because the text is not automatically resolved if the link is used downstream.  
In this particular case, the upstream link text will be used, rather than the topic title specified by the downstream Nebel annotation, but better that than no text, and we can live with the small discrepancy as long as the link works.